sdk/typescript/src/consts.ts

```diff
@@ -14,53 +14,48 @@ export const DefaultSseRetryDurationMs = 1000;
 // #endregion
 
 
-// #region Default strings
-
-
-// #endregion
-
-
 // #region Default booleans
 
 // Should elements be patched using the ViewTransition API?
 export const DefaultElementsUseViewTransitions = false;
 
 // Should a given set of signals patch if they are missing?
-export const DefaultPatchSignalsOnlyIfMissing = false;
+export const DefaultpatchSignalsOnlyIfMissing = false;
```

Comment:
I think a find/replace might have accidentally changed this:
```suggestion
export const DefaultPatchSignalsOnlyIfMissing = false;
```

Reply:
Yes!! Thanks for catching this, fixed!